app/assets/stylesheets/application.scss

```diff
@@ -175,4 +131,26 @@
 }
 .delete-button{
   box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2), 0 6px 20px 0 rgba(0, 0, 0, 0.19);
+}
+.col-sm-6{
+  margin: 0 auto;
+}
```

Comment:
bootstrap overrides unless it's very clear that this is expected consistently and everywhere in the app, should be reserved for rare cases. Try creating a custom class and using that instead